src/amo/components/AddonReviewCard/styles.scss

```diff
@@ -49,6 +49,15 @@
       text-decoration: none;
     }
   }
+
+  .AddonReviewCard-delete:first-child .ConfirmButton-default-button {
+    color: $text-color-default;
+
+    &:active,
+    &:hover {
+      color: $link-color;
```

Comment:
No need to change anything here but I just noticed that `$link-color` is not `$blue-50`. Sigh.

Reply:
I noticed that as well, but figured that `$link-color` was the right thing to use here. Do we want to update what `$link-color` is (in another issue, of course)?

Reply:
Well, I don't know. There are tons of little things like this that are inconsistent between the current site and the mocks. It's a rabbit hole I don't want us to fall into right now. Maybe next quarter.